src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -20,7 +20,7 @@ const AuthenticatorRow = View.extend({
   template: hbs`
     <div class="authenticator-icon-container">
       {{#if logoUri}}
-        <div class="factor-icon authenticator-icon custom-app-logo" role="img"
```

Comment:
I based this refactoring on AuthenticatorVerifyOptions.js, which already has this code logic (you can see the refactor [here](https://github.com/okta/okta-signin-widget/commit/d41b033a33a9c35014efbe82f9c00cc4dee3af6c#diff-bcd9127a5716d292050e226cf7d456d7c3befd3c354dce342dac6debf4eb9be0R20)). I thought that it would be weird for the class name to include "custom-app-logo" if we were using a different authenticator key (like IdP)